### PR TITLE
Extending password settings section

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | master

Extending password settings section.

### CODE REVIEW

File: docs/guide/user_management.md [hunk 1]
```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
-### Setting password expiration
+## Password settings
 
-You can set password expiry rules, which will force users to change their passwords periodically.
+You can customize a password policy in your project.
```
Comment (DominikaK) [inline docs/guide/user_management.md]:
```suggestion
You can customize the password policy in your project.
```

File: docs/guide/user_management.md [hunk 2]
```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
...
+Each password setting is customizable per User Field Type.
+You can change the [password attributes](#password-attributes) or [password expiration settings](#password-expiration), and determine the rules for [repeating passwords](#repeating-passwords).
 
-Password expiry is defined per User Field Type.
-You can set it for the User Content Type in the `ezuser` Field Type's settings:
+To access the settings, perform the following actions:
```
Comment (DominikaK) [inline docs/guide/user_management.md]:
```suggestion
To access the password settings:
```

File: docs/guide/user_management.md [hunk 3]
```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
-### Setting password expiration
+## Password settings
```
Comment (DominikaK) [inline docs/guide/user_management.md]: There are also password settings of a sort in the User menu (change password). Maybe a more unambiguous title?

File: docs/guide/user_management.md [hunk 4]
```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
...
+Each password setting is customizable per User Field Type.
+You can change the [password attributes](#password-attributes) or [password expiration settings](#password-expiration), and determine the rules for [repeating passwords](#repeating-passwords).
 
-Password expiry is defined per User Field Type.
-You can set it for the User Content Type in the `ezuser` Field Type's settings:
+To access the settings, perform the following actions:
+
+1. In the Back Office, in the **Admin** Panel, open the **Content Types** tab.
+1. In the **Content Type Groups** table, click on **Users**.
+1. Edit the User Content Type.
```
Comment (DominikaK) [inline docs/guide/user_management.md]: There can be other CTs that function as users, beyond the built-in User CT (see https://doc.ezplatform.com/en/master/guide/config_repository/#user-identifiers)

File: docs/guide/user_management.md [hunk 5]
```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
...
+- One uppercase letter
+- One lowercase letter
+- One number
+- One non-alphanumeric character
+
+You can also set the minimum password length.
+
+### Password expiration
+
+In the **User account (ezuser)** Field definition, you can set password expiry rules, which will force users to change their passwords periodically.
```
Comment (DominikaK) [inline docs/guide/user_management.md]:
```suggestion
In the **User account (ezuser)** Field definition, you can set password expiration rules, which will force users to change their passwords periodically.
```

File: docs/guide/user_management.md (hunk 3 above)
Comment (michalmilc90) [reply docs/guide/user_management.md]: How about _Password complexity_?

File: docs/guide/user_management.md (hunk 4 above)
Comment (michalmilc90) [reply docs/guide/user_management.md]: Can I paste your comment to the doc? Sounds like ready to be included in this section. :)

File: docs/guide/user_management.md (hunk 3 above)
Comment (DominikaK) [reply docs/guide/user_management.md]: Maybe "Password rules"? So it tries to cover any future policies that may be added.

Comment (michalmilc90) [reply docs/guide/user_management.md]: That's a great idea!

**Pull request merged**